lib/cc/services/github_pull_requests.rb

```diff
@@ -73,4 +89,82 @@ def response_includes_repo_scope?(response)
   def test_status_code
     422
   end
+
+  def welcome_comment_implemented?
+    true
+  end
+
+  def should_post_welcome_comment?
+    config.welcome_comment_enabled && @payload.fetch("first_contribution", false)
+  end
+
+  ALREADY_COMMENTED_PATTERN = /Since this is the first PR/
```

Comment:
Hmm. Don't we externally decide if we've already sent this and set it when sending the service event? Are you worried about race conditions?